src/main/java/com/checkmarx/flow/config/GitHubProperties.java

```diff
@@ -32,9 +32,16 @@ public String getMergeNoteUri(String namespace, String repo, String mergeId){
         //sample: https://api.github.com/repos/octocat/Hello-World/issues/1347/comments
     }
 
-    public String getGitUri(String namespace, String repo){
+    public String getGitUri(String namespace, String repo) {
         String format = "%s/%s/%s.git";
         return String.format(format, getUrl(), namespace, repo);
         //sample: https://github.com/namespace/repo.git
     }
+
+    public String getIssueUri(String namespace, String repo) {
```

Comment:
what is the purpose of this new method. where it is used?

Reply:
yes. now don't use. will remove that